hazelcast/src/main/java/com/hazelcast/instance/GroupProperties.java

```diff
@@ -62,6 +62,18 @@
     public static final String PROP_VERSION_CHECK_ENABLED = "hazelcast.version.check.enabled";
     public static final String PROP_PREFER_IPv4_STACK = "hazelcast.prefer.ipv4.stack";
     public static final String PROP_IO_THREAD_COUNT = "hazelcast.io.thread.count";
+
+    /**
+     * The interval in seconds between {@link com.hazelcast.nio.tcp.handlermigration.IOBalancer IOBalancer}
+     * executions. The shorter intervals will catch I/O Imbalance faster, but they will cause higher overhead.
+     *
+     * Please see documentation of {@link com.hazelcast.nio.tcp.handlermigration.IOBalancer IOBalancer} for
+     * detailed explanation of the problem.
+     *
+     * Default value is 20 seconds. A negative value disables the balancer.
```

Comment:
I don't know how fast an imbalance can occur, be detected and resolved. So should a config value of `0` seconds be a valid interval (from the code it is) or is a minimum of `1` second sufficient?